src/webview/css/css.js

```diff
@@ -455,6 +455,9 @@ body {
 .highlight {
   background-color: hsla(51, 100%, 64%, 0.42);
 }
+[data-mentioned="true"], [data-wildcard_mentioned="true"] {
```

Comment:
We do already have styles for this.
Did you mean to edit these instead of overwriting?